.eslintrc.js

```diff
@@ -3,6 +3,6 @@ module.exports = {
     browser: true
   },
   plugins: ['jsdoc'],
-  extends: 'eslint-config-digitalbazaar',
+  extends: 'eslint-config-digitalbazaar/jsdoc',
```

Comment:
This eslint-config name breaks the pattern I conceived in my mind. It seems the pattern was `eslint-config-digitalbazaar/<env>`, eg. **vue:** `eslint-config-digitalbazaar/vue`. This does not block this PR.

Reply:
I think the pattern is that the thing at the end is the plugin, not the environment.

Reply:
So if we wanted to use jsdocs in our vue environment then it would be...

```
extends: ['eslint-config-digitalbazaar/vue','eslint-config-digitalbazaar/jsdoc']
```

Is the above statement correct?

Reply:
I'm very pleased that you have brought up this topic.  What you are suggesting is the way I think it should work, except, maybe you would actually go one step further:
```
extends: ['eslint-config-digitalbazaar', 'eslint-config-digitalbazaar/vue','eslint-config-digitalbazaar/jsdoc']
```
keep in mind here that these *could* be shortened to just digitalbazaar, digitalbazaar/vue.  Somewhere along the line it was decided that we would use the verbose full names.

At any rate, the way the `vue` and `jsdoc` rules are currently setup do not accurately reflect what I have written above.  `vue` and `jsdoc` already extend the base linter eslint-config-digitalbazaar as shown here (the base rules are represented here by index.js): https://github.com/digitalbazaar/eslint-config-digitalbazaar/blob/master/vue.js#L5

So, I don't know if the current rule setup causes any problems or confusion on the part of the linter.  Also, since eslint rules are additive,  it's common that we would have eslint-config-digitalbazaar at the root of the project, then eslint-config-digitalbazaar/vue in the components folder. 

So, thus far, just using 'eslint-config-digitalbazaar-vue' has just been working the way people expect, without having the be explicit about adding the base 'eslint-config-digitalbazaar' rules.

Reply:
why don't we move this discussion to eslint-config-digitalbazaar maybe we could make an issue seperate rules so we can extened by composition i.e.

extends: ['digitalbazzar', 'digitalbazaar/vue'] etc.

something tells me this Pr is about to get pretty big.